ui/app/src/components/Toast.tsx

```diff
@@ -0,0 +1,45 @@
+// Copyright 2021 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Alert, Snackbar } from '@mui/material';
+import { SyntheticEvent, useEffect, useState } from 'react';
+
+export interface ToastProperties {
+  loading: boolean;
+  message?: string;
+  severity: 'error' | 'warning' | 'info' | 'success';
+}
+
+export default function Toast(props: ToastProperties) {
```

Comment:
👏

Reply:
I have some more "common" code here that I might be able to contribute to the project once I talk to Rob around showing errors via a Toast globally. Will follow up.

Reply:
yeah that would be great to have a common way to handle the error. I think here it's a bit too local and doesn't really scale